### DISCUSSION TITLE
v1.5.3 Optional Release and Discussion

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#5846] [Announcement]
[v1.5.3](https://github.com/filecoin-project/lotus/releases/tag/v1.5.3) is out.

There are some small changes you may wanna know about the miner dealmaking configuration and adjust your setup accordingly, for more details, so the configuration docs [here](https://docs.filecoin.io/mine/lotus/miner-configuration/#dealmaking-section).

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
ran several daemons and a miner with it for multiple days without problems - so far.

@jennijuju can you guys specify what exactly NEEDS (in the sense of: if you update A then you NEED to update B) to be updated in the `lotus <--> lotus-miner <--> lotus-worker` chain of processes, depending on each other, when you do a release?

**Comment by cwhiggins** [comment]
running v1.5.3-rc2 for two days. Yesterdays Post almost failed as sync really slowed down during an offline deal import. Batch publishing deals results in all deals in a batch then immediately beginning to seal after deal is published, no more wait delay, which can be problematic.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
```
Magik6k  1 day ago
@Patrick - Factor8 Solutions with the recent changes the miner will create up to MaxWaitDealsSectors and put deals into them in parallel. If you want to limit that, just set that value lower
```
